cla_backend/translations/en_GB/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,2213 @@
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:1528
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:1573
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:1618
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:1664
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:1834
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:1880
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:1968
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2013
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2058
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2169
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2214
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2364
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2411
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2460
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2680
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2726
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2772
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2818
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2864
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2910
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:2956
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3029
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3075
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3121
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3239
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3285
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3331
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3377
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3423
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3538
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3705
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:3751
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:4515
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:4943
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:4991
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:5205
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:5230
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:5346
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:5392
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:5458
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:5550
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:5667
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:5713
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:5759
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:5805
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:6239
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:6474
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:7453
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:7889
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:7939
+#: apps/diagnosis/data/graph-2015.06.10.graphml.tpl:8015
+msgid "INSCOPE"
```

Comment:
Should definitely not be translated in the template. Could have a list of 'non translated constants/words' that don't get wrapped in translation tags or something